packages/frontend/core/src/components/app-sidebar/quick-search-input/index.css.ts

```diff
@@ -28,3 +28,9 @@ export const shortcutHint = style({
   color: cssVarV2('text/tertiary'),
   fontSize: cssVar('fontBase'),
 });
+export const quickSearchBarEllipsisStyle = style({
+  maxWidth: '200px',
```

Comment:
```suggestion
```
Remove this

Reply:
and add `overflow: 'hidden',` to `root` class
```
export const root = style({
...
  overflow: 'hidden',
});
```

Reply:
You’re right about the maxWidth, but after setting `overflow: 'hidden'` on the root, it now looks like this. Is it okay?

![image](https://github.com/user-attachments/assets/891d1021-4e6d-4e16-829a-3d145984e43c)

Reply:
You were absolutely right about the maxWidth. I realise now that it wasn’t necessary at all. I’m not sure what I was thinking at the time. Great code review, much appreciated @JimmFly